src/Movie/Movie.js

```diff
@@ -0,0 +1,38 @@
+import './Movie.css'
+
+const Movie = ({ movieDetails }) => {
+    const {
+        title, posterPath, backdropPath, releaseDate, overview,
+        genres, budget, revenue, runtime, tagline, averageRating
+    } = movieDetails;
+
+    const releaseDateFormatted = new Date(releaseDate).toLocaleDateString('en-US');
+    
+    return (
+        <div className='movie-container'>
+            <div className='movie-heading-container' 
+                style={{backgroundImage: `url(${backdropPath})`, 
+                backgroundSize: 'cover',
+                boxShadow: 'inset 30vw 10px 100px black'
```

Comment:
are these the inline styles you were telling me about? I love it should I do that on my components too for consistency?